nitric/proto/deployments/v1/deployments.proto

```diff
@@ -161,6 +161,31 @@ message Service {
   map<string, string> env = 14; 
 }
 
+message Job {
+  // The name of the job to create
+  string name = 1;
```

Comment:
Do we need job queue name here as well? Or are we handling that.

Reply:
This is deploy time, this is like naming a topic then publishing to that topic using it's name.